Modules/arraymodule.c

```diff
@@ -2873,7 +2873,7 @@ array_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
         }
     }
     PyErr_SetString(PyExc_ValueError,
-        "bad typecode (must be b, B, u, h, H, i, I, l, L, q, Q, f or d)");
+        "bad typecode (must be b, B, u, w, h, H, i, I, l, L, q, Q, f or d)");
```

Comment:
```suggestion
        "bad typecode (must be b, B, u, w, h, H, i, I, l, L, q, Q, f or d)");
```
Ordering is and was questionable, maybe there is some sense to this? @tomasr8 maybe you know?